bin/dry-run.js

```diff
@@ -1,3 +1,6 @@
+// IMPORTANT! We must run dotenv config as the very first thing before importing anything else
```

Comment:
```suggestion
```

I don't like when me code SHOUTS at me 😅 Would you be ok with either option?

```suggestion
// dotenv has to be imported before anything else
```

Reply:
I am going to use this:

```js
// dotenv must be imported before importing anything else
```